test/clt-tests/core/show-threads.rec

```diff
@@ -0,0 +1,16 @@
+––– block: ../base/start-searchd-1 –––
+––– input –––
+mysql -h0 -P9306 -e "SHOW THREADS\G;"
+––– output –––
+*************************** 1. row ***************************
+TID: %{NUMBER}
+Name: work_%{NUMBER}
+Proto: mysql
+State: query
+Connection from: 127.0.0.1:%{NUMBER}
+ConnID: %{NUMBER}
+This/prev job time, s: %{NUMBER}ms
```

Comment:
Pls check with Dmitrii on this. As we see in the test results, the actual title doesn't include `, s`, but the test passes.